src/main/java/org/terasology/climateConditions/HypothermiaSystem.java

```diff
@@ -27,9 +27,14 @@
 import org.terasology.logic.players.PlayerCharacterComponent;
 import org.terasology.physics.events.MovedEvent;
 
+/**
+ * This system is responsible for adding a Hypothermia Component to the player whenever a player enters the danger
+ * zone(extreme cold climate). And also for modifying the speed of the player as long as the player has a Hypothermia
+ * Component.
+ */
 @RegisterSystem(value = RegisterMode.AUTHORITY)
 public class HypothermiaSystem extends BaseComponentSystem {
-    private final float thresholdHeight = 60f;
+    private float thresholdHeight = 600f;
```

Comment:
I guess that's a temporary alteration for testing? I'd suggest to remove it and from now on not staging and checking in the file.

Reply:
Oops  I'll open a different PR...😅😅😅